src/run.sh

```diff
@@ -15,5 +15,4 @@ else
   fi
 fi
 
-python manage.py collectstatic --settings=registrar.config.settings --noinput
```

Comment:
Maybe it's worth replacing this command with a comment that explains that the site will only work right if `collectstatic` has been run somewhere else, just as a breadcrumb for the future?